presto-main/src/main/java/com/facebook/presto/operator/HashAggregationOperator.java

```diff
@@ -511,8 +516,8 @@ private void closeAggregationBuilder()
             // The reference must be set to null afterwards to avoid unaccounted memory.
             aggregationBuilder = null;
         }
-        operatorContext.setMemoryReservation(0);
-        operatorContext.setRevocableMemoryReservation(0);
+        localUserMemoryContext.setBytes(0);
+        localRevocableMemoryContext.setBytes(0);
```

Comment:
`close`?

Reply:
`closeAggregationBuilder` can be called multiple times so we cannot close here as subsequent updates to these contexts will fail.